src/cpp/server/db/20200226141952248_AddRevokedCookie.sql

```diff
@@ -0,0 +1,19 @@
+/* Stores revoked auth cookies - cookies that are not yet expired, but
```

Comment:
Can we add a README to this parent folder that explains how to name files and perform schema upgrades for devs?